src/sass/components/_feature.scss

```diff
@@ -2,40 +2,51 @@
 // SPDX-License-Identifier: BSD-3-Clause
 
 .#{$prefix}-feature {
-  max-width: 100%;
-
-  &__image {
-    overflow: hidden;
-  }
-
   & img {
```

Comment:
Since you're already wrapping the image in a div with the `rvt-feature__image` class. I think it makes more sense to move these declarations to a block using that class as a selctor. So instead of the more generic `img` selector it would become.

```scss
&__image img {
  display: block;
  width: 100%;
}
```

That way you're only targeting an `img` element within the wrapper instead of any image in the whole component, i.e. `.rvt-feature img`, which would target _any_ `img` that is a child of an element with `.rvt-feature` applied to it. Not that there _should_ be more images outside the `__image` wrapper, but just to be safe it makes sense to scope the styles you're writing with a little more intention in this case.